editor/editor_run_native.cpp

```diff
@@ -38,81 +38,59 @@
 void EditorRunNative::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_ENTER_TREE: {
-			for (int i = 0; i < EditorExport::get_singleton()->get_export_platform_count(); i++) {
-				Ref<EditorExportPlatform> eep = EditorExport::get_singleton()->get_export_platform(i);
-				if (eep.is_null()) {
-					continue;
-				}
-				Ref<ImageTexture> icon = eep->get_run_icon();
-				if (!icon.is_null()) {
-					Ref<Image> im = icon->get_image();
-					im = im->duplicate();
-					im->clear_mipmaps();
-					if (!im->is_empty()) {
-						im->resize(16 * EDSCALE, 16 * EDSCALE);
-						Ref<ImageTexture> small_icon = ImageTexture::create_from_image(im);
-						MenuButton *mb = memnew(MenuButton);
-						mb->get_popup()->connect("id_pressed", callable_mp(this, &EditorRunNative::run_native).bind(i));
-						mb->connect("pressed", callable_mp(this, &EditorRunNative::run_native).bind(-1, i));
-						mb->set_icon(small_icon);
-						add_child(mb);
-						menus[i] = mb;
-					}
-				}
+			remote_debug = memnew(MenuButton);
+			remote_debug->get_popup()->connect("id_pressed", callable_mp(this, &EditorRunNative::run_native));
+			remote_debug->set_icon(get_theme_icon(SNAME("PlayRemote"), SNAME("EditorIcons")));
+			remote_debug->set_tooltip_text(TTR("Remote Debug"));
+			add_child(remote_debug);
+		} break;
+		case NOTIFICATION_THEME_CHANGED: {
+			if (remote_debug) {
+				remote_debug->set_icon(get_theme_icon(SNAME("PlayRemote"), SNAME("EditorIcons")));
 			}
 		} break;
-
```

Comment:
Tiny nitpick, but the empty line between `case`s in `_notification` switches is intentional. Those `case`s are often pretty big, function-like blocks so have one extra line between those blocks helps with readability (notably to read the `NOTIFICATION_*` value after `case` in my opinion.